raiden-ts/src/channels/epics/block.ts

```diff
@@ -184,6 +186,37 @@ export function blockStaleEpic(
   );
 }
 
+/**
+ * Fetch settleTimeout from contract once
+ *
+ * @param action$ - Observable of RaidenActions
+ * @param state$ - Observable of RaidenStates
+ * @param deps - RaidenEpicDeps members
+ * @param deps.registryContract - TokenNetworkRegistry instance
+ * @param deps.config$ - Config observable
+ * @param deps.init$ - Observable of sync tasks
+ * @returns Observable of contractSettleTimeout actions
+ */
+export function contractSettleTimeoutEpic(
+  {}: Observable<RaidenAction>,
+  {}: Observable<RaidenState>,
+  { registryContract, config$, init$ }: RaidenEpicDeps,
+): Observable<contractSettleTimeout> {
+  let done$: AsyncSubject<null>;
+  return defer(async () => {
+    done$ = new AsyncSubject();
+    init$.next(done$);
+    return registryContract.callStatic.settle_timeout();
```

Comment:
The only way for a `TokenNetwork` to be constructed is via Registry. There are already several places where we assume this, this is per contract construction, and I'm pretty sure the clients nor the services could work with contracts which don't come from the registry. If we want to support this kind of environment, we would need to state this and justify why this could be useful, but right now, it's not possible, and therefore I'd say we don't need/want to code for this case and can safely assume all TNs have the same `settle_timeout`, registry's.

Reply:
Alright. If this is a general assumption in the whole SDK atm, I'm fine with that.
We do it differently in the services, but don't that matters.